doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -68,24 +71,27 @@ Working with nRF54L Series
 Working with nRF53 Series
 =========================
 
+* Restructured the contents of this section into multiple pages, with :ref:`ug_nrf53` as the landing page.
+* Added the :ref:`features_nrf53` page.
 * Moved :ref:`ug_nrf5340_gs` to the :ref:`gsg_guides` section.
+* Changed the default value for the Kconfig option :kconfig:option:`CONFIG_CLOCK_CONTROL_NRF_ACCURACY` from 500 to 250 if :kconfig:option:`CONFIG_CLOCK_CONTROL_NRF_K32SRC_RC` is used.
```

Comment:
```suggestion
* Changed the default value for the Kconfig option :kconfig:option:`CONFIG_CLOCK_CONTROL_NRF_ACCURACY` from ``500`` to ``250`` if :kconfig:option:`CONFIG_CLOCK_CONTROL_NRF_K32SRC_RC` is used.
```

These sentences (under Working with nRF53 Series) could be in the Added and updated bullet points.